### PR TITLE
Add Dustmaps

### PR DESCRIPTION
**Pull request by guillochon**
Recipe for the dustmaps package

### CODE REVIEW

File: recipes/dustmaps/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  build:
+    - python
+    - pip
+    - numpy
+    - scipy
+    - astropy
+    - h5py
+    - healpy
+    - requests
+    - progressbar2
```
Comment (CJ-Wright) [inline recipes/dustmaps/meta.yaml]: Are all of these build time requirements?

File: recipes/dustmaps/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+{% set name = "dustmaps" %}
+{% set version = "0.1a12" %}
```
Comment (CJ-Wright) [inline recipes/dustmaps/meta.yaml]: Would it be possible to not build from an alpha release?

File: recipes/dustmaps/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  license_file: LICENSE.txt
+  summary: 'A uniform interface for a number of 2D and 3D maps of interstellar dust reddening/extinction.'
+
+  doc_url: http://dustmaps.readthedocs.io/
+  dev_url: https://github.com/gregreen/dustmaps
+
+extra:
+  recipe-maintainers:
+    - guillochon
+    - pkgw
```
Comment (CJ-Wright) [inline recipes/dustmaps/meta.yaml]: @pkgw are you interested in being a maintainer?

File: recipes/dustmaps/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    - gregreen
```
Comment (CJ-Wright) [inline recipes/dustmaps/meta.yaml]: @gregreen are you interested in being a maintainer?

File: recipes/dustmaps/meta.yaml (hunk 2 above)
Comment (guillochon) [reply recipes/dustmaps/meta.yaml]: Only if @gregreen releases one, but the package is functional.

Comment (gregreen) [reply recipes/dustmaps/meta.yaml]: I've been calling every release up until now "alpha." That's probably being conservative at this point though.

Comment (guillochon) [reply recipes/dustmaps/meta.yaml]: Seems fine to me to just push it out as `0.1`, which indicates that it's an unfinished state. Unless you think it's finished, in which case I'd suggest `1.0`...

**Comment by marcelotrevisani** [comment]
ping @guillochon

File: recipes/dustmaps/meta.yaml (hunk 3 above)
Comment (pkgw) [reply recipes/dustmaps/meta.yaml]: Whoops, this got a bit stale! Yes, I am.

**Comment by wkerzendorf** [comment]
I am just sitting here with @gregreen and contemplating adding dustmaps to conda-forge. But @guillochon is already doing it. @guillochon do you need help?

File: recipes/dustmaps/meta.yaml (hunk 4 above)
Comment (gregreen) [reply recipes/dustmaps/meta.yaml]: I'd rather not, as I don't normally use conda. Thanks for packaging dustmaps, though!

**Comment by guillochon** [comment]
@conda-forge-admin, please rerender

**Comment by guillochon** [comment]
@pkgw Should be, yes.

File: recipes/dustmaps/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
+{% set name = "dustmaps" %}
+{% set version = "1.0" %}
+{% set sha256 = "f9099f12b84aa28959b171ea9057f178511595c50db9e3b1775e1fb924910108" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (pkgw) [inline recipes/dustmaps/meta.yaml]: The `fn` item is no longer needed

File: recipes/dustmaps/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: True  # [win]
```
Comment (pkgw) [inline recipes/dustmaps/meta.yaml]: Does this actually need to be skipped on Windows? Can this be `noarch: python`? [Cf docs](https://conda-forge.org/docs/meta.html#building-noarch-packages).

File: recipes/dustmaps/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (pkgw) [inline recipes/dustmaps/meta.yaml]: The preferred magic install line is now:

```
script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```


([Cf docs](https://conda-forge.org/docs/meta.html#use-pip))

File: recipes/dustmaps/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+  summary: 'A uniform interface for a number of 2D and 3D maps of interstellar dust reddening/extinction.'
+
+  doc_url: http://dustmaps.readthedocs.io/
+  dev_url: https://github.com/gregreen/dustmaps
+
+extra:
+  recipe-maintainers:
+    - guillochon
+    - pkgw
+    - gregreen
```
Comment (pkgw) [inline recipes/dustmaps/meta.yaml]: I think @gregreen said he wanted to be left off the maintainer list.

**Comment by guillochon** [comment]
Build failing now @pkgw, any thoughts as to why?

**Comment by guillochon** [comment]
@pkgw I think we're good now, mind checking again?

**Pull request merged**